sdk/src/types/block/output/feature/block_issuer.rs

```diff
@@ -244,6 +245,12 @@ impl StorageScore for BlockIssuerFeature {
     }
 }
 
+impl WorkScore for BlockIssuerFeature {
+    fn work_score(&self, params: WorkScoreParameters) -> u32 {
+        params.block_issuer()
```

Comment:
This needs some extra multiplication with the amount of keys?
https://github.com/iotaledger/tips/blob/tip42/tips/TIP-0042/tip-0042.md#storage-score

Reply:
storage score?

Reply:
Ah sorry :joy: